tests/SharpCompress.Test/SharpCompress.Test.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
Please change this back for now.  I haven't updated things across the board to 3.1 yet.

Reply:
Sorry, had to do this to get it running on my machine.